addons/mail/static/src/discuss/core/public_web/discuss_patch.js

```diff
@@ -12,4 +12,8 @@ patch(Discuss.prototype, {
         super.setup(...arguments);
         this.rtc = useService("discuss.rtc");
     },
+
+    get showCallView() {
+        return !this.rtc.state.isFullscreen && this.thread.rtc_session_ids.length > 0;
+    },
```

Comment:
Exact same getter already defined in `call/common`, so drop this one in `core/public_web`